metrics/telemetry.go

```diff
@@ -269,6 +276,7 @@ type TablePartitionUsageCounter struct {
 	TablePartitionAddIntervalPartitionsCnt    int64 `json:"table_partition_add_interval_partitions_cnt"`
 	TablePartitionDropIntervalPartitionsCnt   int64 `json:"table_partition_drop_interval_partitions_cnt"`
 	TablePartitionComactCnt                   int64 `json:"table_TablePartitionComactCnt"`
```

Comment:
(Not really part of this PR) Can this be changed to TablePartitionCompactCnt / table_TablePartitionComactCnt (adding a 'p' Comact -> Compact)? Since the json is already defined.

Reply:
I guess It is possible. But you need to contact the telemetry(observe) team to change the column name, and change how their HTTP api handles telemetry requests.